test/e2e/framework/test_context.go

```diff
@@ -292,6 +295,8 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.BoolVar(&TestContext.ListImages, "list-images", false, "If true, will show list of images used for runnning tests.")
 	flags.StringVar(&TestContext.KubectlPath, "kubectl-path", "kubectl", "The kubectl binary to use. For development, you might use 'cluster/kubectl.sh' here.")
+
+	flags.StringVar(&TestContext.ProgressUpdateURL, "progress-report-url", "", "The URL to POST progress updates to as the suite runs to assist in aiding integrations. If empty, no messages sent.")
```

Comment:
nit: just a question.
Why don't we name `ProgressReportURL` instead?
Most test context variables consist of the same words from the option names like the above.
That could be easy to search the code from the option name.